tests/unit/multi.tcl

```diff
@@ -121,6 +121,18 @@ start_server {tags {"multi"}} {
         r exec
     } {}
 
+    test {EXEC fail on WATCHed key modified (swapdb)} {
+        r select 0
+        r set k 1
+        r select 1
+        r set k 2
+        r watch k
+        r swapdb 0 1
+        r multi
+        r get k
+        r exec
+    } {}
+
```

Comment:
do we still need this test now? or is it fully covered by the other two?

Reply:
removed